packages/code-editor/src/Markdown/Markdown.spec.tsx

```diff
@@ -28,7 +28,7 @@ import React from 'react'
 import { screen } from '@testing-library/react'
 import { renderWithTheme } from '@looker/components-test-utils'
 
-import { renderWithSearch } from '../../test-utils'
+import { renderWithSearch } from '../../../api-explorer/src/test-utils'
```

Comment:
yup. imports get a bit ugly here. Not your fault, just something I noticed when I first started playing with moving this component.